plugins/Jobs.cpp

```diff
@@ -268,18 +279,33 @@ bool BedrockPlugin_Jobs::processCommand(BedrockNode* node, SQLite& db, BedrockNo
             throw "402 Invalid priority value";
         }
 
-        // Create this new job
-        db.write("INSERT INTO jobs ( created, state, name, nextRun, repeat, data, priority, parentJobID ) "
-                 "VALUES( " +
-                    SCURRENT_TIMESTAMP() + ", " + 
-                    SQ("QUEUED") + ", " + 
-                    SQ(request["name"]) + ", " + 
-                    safeFirstRun + ", " +
-                    SQ(SToUpper(request["repeat"])) + ", " + 
-                    safeData + ", " + 
-                    SQ(priority) + ", " + 
-                    safeParentJobID +
-                 " );");
+        if (mergeIntoID) {
+            // Update the existing job.
+            if(!db.write("UPDATE JOBS SET "
```

Comment:
I think we should keep the old one personally. Also, I'm not sure why we would want to update the repeat or the priority either. I think we should only merge the data. 

My reasoning is that I think that the first time a unique job is created, it's settings should trump all other attempts to create it. This is the most consistent with the current functionality. Otherwise, we are just doing a lookup for an existing job and updating it, which isn't an upsert.